qa/common/shm_util.py

```diff
@@ -441,6 +441,9 @@ def __exit__(self, type, value, traceback):
                     print(
                         f"Shared memory leak detected [{shm_region}]: {curr_shm_free_size} (curr free) < {prev_shm_free_size} (prev free)."
                     )
+                    # Known shared memory leak of 480 bytes in BLS test.
```

Comment:
Do we have any ticket to eventually find/resolve this issue all together? Would be good to leave the FIXME `DLIS-XXXX` here if so.

Reply:
Good call, added the ticket reference.